pkg/virt-controller/services/template_test.go

```diff
@@ -354,6 +354,7 @@ var _ = Describe("Template", func() {
 								DisableHotplug: true,
 							},
 						},
+						Architecture: arch,
```

Comment:
I think you'll need to change L389 in the test to expect `"kubernetes.io/arch": <arch>` in the nodeselector.
```
			Expect(pod.Spec.NodeSelector).To(Equal(map[string]string{
					v1.NodeSchedulable: "true",
                        }))
```

Reply:
Will update this, thanks